FEM/pcs_dm.cpp

```diff
@@ -772,19 +772,21 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				if (Norm < Tolerance_global_Newton && Error > Norm)
```

Comment:
@norihiro-w do you have a better convergence handling for deformation problems somewhere in your branches? I imagine a separate control of absolute and/or relative displacement and force tolerances, i.e. 4 convergence criteria.